RetailCoder.VBE/Inspections/QuickFixes/AssignedByValParameterMakeLocalCopyQuickFix.cs

```diff
@@ -49,7 +49,7 @@ public override void Fix()
 
         private void RequestLocalCopyVariableName()
         {
-            using( var view = _factory.Create(_target.IdentifierName, _target.DeclarationType.ToString()))
+            using( var view = _dialogFactory.Create(_target.IdentifierName, _target.DeclarationType.ToString()))
```

Comment:
@BZngr ah, sorry I missed that! don't bother setting up a mock for a test that doesn't need it ;-)

I guess a mock will be useful for a test that tests the quickfix itself.